std/concurrency.d

```diff
@@ -1422,57 +1333,56 @@ private:
         void switchContext() nothrow
         {
             mutex_nothrow.unlock_nothrow();
-            scope(exit) mutex_nothrow.lock_nothrow();
+            scope (exit) mutex_nothrow.lock_nothrow();
```

Comment:
There are 271 occurrences of `scope(exit)` in Phobos, so maybe we should agree on a common style here as well?